app/controllers/stories_controller.rb

```diff
@@ -196,6 +184,18 @@ def handle_article_show
     render template: "articles/show"
   end
 
+  def assign_feed_stories
+    feed = Articles::Feed.new(number_of_articles: 35, page: @page, tag: params[:tag])
+    if %w[week month year infinity].include?(params[:timeframe])
```

Comment:
Seeing this here again makes me think that maybe there is a point to be made for the service object approach. Instead of forcing one helper method to accommodate both use cases, we could have something like `feed.stories` and `feed.feed` (`feed.feed_stories`, I haven't had enough ☕️ for sensible naming yet). But again, feel free to ignore this, we can always revisit it later.

Reply:
See my comment here :https://github.com/thepracticaldev/dev.to/pull/6070#discussion_r380760041, but I agree I'd like to get this logic into `Articles::Feed`, it's just too different in the two places it's currently used IMHO